rules/windows/builtin/win_net_ntlm_downgrade.yml

```diff
@@ -1,4 +1,3 @@
-action: global
```

Comment:
Is there a reason it should be removed?

Reply:
Not anymore, I just tried again and I don't have the conversion error that was causing this request. I am closing the PR and the associated issue. Thanks for the question 👍